### PR TITLE
Fix undo/redo bug

### PR DESCRIPTION
**Pull request by erisjacey**
- Participate/presence now responsive when undo-ing

### CODE REVIEW

File: src/main/java/atas/model/session/VersionedAttributesList.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        return currentAttributeList;
+    }
+    public void setCurrentAttributeList(List<Attributes> attributeList) {
+        this.currentAttributeList.setAll(attributeList);
+    }
+    private void resetData(List<Attributes> newData) {
+        this.currentAttributeList.setAll(newData);
+    }
+    private void removeStatesAfterCurrentPointer() {
+        this.attributeStateList.subList(currentStatePointer + 1, attributeStateList.size()).clear();
```
Comment (nweiyue) [inline src/main/java/atas/model/session/VersionedAttributesList.java]: Perhaps breaking this line of code can improve readability?
e.g., 
List<Attributes> sublist = this.attributeStateList.subList(currentStatePointer + 1, attributeStateList.size());
sublist.clear()

**Review by nweiyue — APPROVED** [review]
LGTM! Just a nitpick.

File: src/main/java/atas/model/session/VersionedAttributesList.java (hunk 1 above)
Comment (erisjacey) [reply src/main/java/atas/model/session/VersionedAttributesList.java]: Ok thanks, will do!

**Review by marcustw — APPROVED** [review]
LGTM!

**Pull request merged**